static/app/data/forms/organizationGeneralSettings.tsx

```diff
@@ -99,12 +99,6 @@ const formGroups: JsonFormObject[] = [
           'Allow organization members to invite other members via email without needing org owner or manager approval.'
         ),
         visible: ({features}) => features.has('members-invite-teammates'),
-        disabled: ({features, access}) =>
-          !access.has('org:write') || !features.has('team-roles'),
```

Comment:
access.has(org:write) should still be there right? We don't want members to be able to toggle.

Reply:
I think I actually added this unnecessarily the first time, since none of the other settings here have this condition. I believe this entire settings page is already disabled for non owner/managers